base/runtime/internal.odin

```diff
@@ -1106,3 +1106,12 @@ __read_bits :: proc "contextless" (dst, src: [^]byte, offset: uintptr, size: uin
 		dst[j>>3]  |= the_bit<<(j&7)
 	}
 }
+
+@(no_sanitize_address)
+__asan_unpoison_memory_region :: #force_inline proc "contextless" (address: rawptr, size: uint) {
+	foreign {
+		__asan_unpoison_memory_region :: proc "system" (address: rawptr, size: uint) ---
+	}
+	__asan_unpoison_memory_region(address, size)
+}
```

Comment:
Why is this wrapped?

Reply:
I wasn't sure and didn't try to find out if `lb_emit_runtime_call` could directly call foreign procedures.

Reply:
Turns out it is not needed.